.github/workflows/build.yml

```diff
@@ -0,0 +1,29 @@
+name: TDR Tag and pre deploy
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - main
+jobs:
+  pre-deploy:
+    uses: nationalarchives/tdr-github-actions/.github/workflows/lambda_build.yml@main
+    with:
+      repo-name: tdr-file-upload-data
```

Comment:
Is that the correct `repo-name` value?

Reply:
It's not, copy and paste errors strike again.